lib/__tests__/ignore.test.js

```diff
@@ -327,12 +327,7 @@ describe("using ignoreFiles with input files that would cause a postcss syntax e
   });
 
   it("syntax-error-ignored.scss found", () => {
-    // For node 8 on Windows
-    const index = Number(
-      results[1].source.indexOf("syntax-error-ignored.scss")
-    );
-
-    expect(index).not.toBe(-1);
+    expect(results[1].source.indexOf("syntax-error-ignored.scss")).not.toBe(-1);
```

Comment:
/cc @hudochenkov due this lines we have failed CI on windows at random time, i don't investigate why it is happens, but this change fix problem

Reply:
We did have exactly this and had random fails as well https://github.com/stylelint/stylelint/pull/3810. I think we can use:

```js
expect(results[1].source.includes("syntax-error-ignored.scss")).toBe(true);
```